pkg/controller/podautoscaler/metrics/client.go

```diff
@@ -104,10 +101,10 @@ func getContainerMetrics(rawMetrics []metricsapi.PodMetrics, resource v1.Resourc
 			}
 		}
 		if !containerFound {
-			return nil, fmt.Errorf("container %s not present in metrics for pod %s/%s", container, m.Namespace, m.Name)
+			klog.FromContext(ctx).V(2).Info("Missing container metric", "container", container, "namespace", m.Namespace, "pod", m.Name)
```

Comment:
nit: what do you think about making this a warning?

This isn't an expected steady-state thing so perhaps we can make this more visible in the logs incase someone isn't realizing a container is crash looping?

Reply:
I don't really have a strong opinion here other than I used the same log level we are currently using in getPodMetrics.

https://github.com/kubernetes/kubernetes/blob/0c5e832aa2bb32573ca695858bc421d111c90119/pkg/controller/podautoscaler/metrics/client.go#L122

If we do update it, we should probably do it in both places

Reply:
It also looks like this package only has `info` and `error`. I can adjust the verbosity int here; but will probably need a pointer to what each levels mean in this repo

Reply:
FYI - https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md